test/e2e/install/install_test.go

```diff
@@ -207,15 +207,16 @@ func installMayastor() {
 
 	fmt.Printf("registry address %v, number of mayastor instances=%v\n", registryAddress, numMayastorInstances)
 
+	// FIXME use absolute paths, do not depend on CWD
```

Comment:
Yes, please, I would suggest doing it even now, with something like

```
MAIN_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
```

at the beginning and referencing everything from that.

Reply:
We've agreed to address this throughout whole tests in a later PR.